lib/ASTGen/Package.swift

```diff
@@ -0,0 +1,32 @@
+// swift-tools-version: 5.6
+
+import PackageDescription
+
+let package = Package(
```

Comment:
Would it make sense to add this paragraph of yours to Package.swift?
> The CMake build system is the only one that's able to produce a working compiler. There is also a Package.swift that makes it easier to build and work with the swiftASTGen library within IDEs, but it's mainly there for editing---it won't create something that can be meaningfully executed. Most things with the new Swift parser are better implemented/tested within or on top of the swift-syntax package.

Reply:
Great idea!